pkg/cli/demo_locality_test.go

```diff
@@ -14,12 +14,19 @@
 package cli
 
 import (
+	"fmt"
+
 	"github.com/cockroachdb/cockroach/pkg/cli/democluster"
 	"github.com/cockroachdb/cockroach/pkg/security/securityassets"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
 )
 
 func Example_demo_locality() {
+	_ = disabledExample_demo_locality
+	fmt.Println("skipped due to flake: https://github.com/cockroachdb/cockroach/issues/99903")
```

Comment:
is this better than just renaming the func `skippedExample....` ?

Reply:
lol i only saw your message afterwards. sorry about that

Reply:
I dismissed it right after I posted -- decided it wasn't worth waiting for.